### PR TITLE
fix: finetune schemas and add geometry computed column

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: .postgresql/drop tables.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DROP TABLE "Integration"."Addresses";
```
Comment (ArneD) [inline .postgresql/drop tables.sql:1]: for dev purposes?

File: src/Basisregisters.IntegrationDb.Schema/IntegrationContext.cs [hunk 2]
```diff
@@ -11,8 +11,14 @@ public class IntegrationContext : DbContext
     {
         public const string Schema = "Integration";
         public const string MigrationsTableName = "__EFMigrationsHistoryIntegration";
+        public const string GmlComputedValueQuery = "ST_GeomFromGML(REPLACE(\"GeometryGml\",'https://www.opengis.net/def/crs/EPSG/0/', 'EPSG:')) ";
```
Comment (ArneD) [inline src/Basisregisters.IntegrationDb.Schema/IntegrationContext.cs:14]: 👍

**Review by ArneD — APPROVED** [review]

File: .postgresql/drop tables.sql (hunk 1 above)
Comment (pgallik) [reply .postgresql/drop tables.sql:1]: yes

**Pull request merged**